### ISSUE TITLE
Schema limits HTTP Compress bufferSize to smaller max than permitted on BIG-IP

### ISSUE THREAD

**Issue by MicSkr** [F5Networks/f5-appsvcs-extension#284]
### Environment
 * Application Services Version: 3.18.0
 * BIG-IP Version: 14.1.2.3

### Summary
AS3 Schema doesn't permit what's allowed in BIG-IP. HTTP Compression buffer size in BIG-IP permits buffer size 0 - 4294967295 ... AS3 Schema limits at 32768. Prohibits migrating large value buffersizes to AS3 model.

AS3 HTTP_Compress Schema Link:
https://clouddocs.f5.com/products/extensions/f5-appsvcs-extension/3.19/refguide/schema-reference.html#http-compress 

...
bufferSize (integer) 	4096 	[256, 32768] 	Maximum number of response octets to buffer before deciding whether to apply compression (default 4096)
...

### Steps To Reproduce
1. Submit the following declaration:
```json
    {
        "class": "AS3",
        "action": "deploy",
        "declaration": {
            "class": "ADC",
            "schemaVersion": "3.18.0",
            "target": {
                "address": "10.1.1.10 "
            },
            "app_1": {
                "class": "Tenant",
                "app_1": {
                    "class": "Application",

                    "template": "http",
                    "serviceMain": {
                        "virtualPort": 80,

                        "virtualAddresses": [
                            "10.1.10.101"
                        ],
                        "pool": "Pool",
                        "enable": true,
                        
                        "iRules": [],
                        "class": "Service_HTTP",
                        "profileHTTPCompression":{
                            "use": "compress_profile"
                        }
                        
                    },

                    "Pool": {
                        "members": [
                            {
                                "servicePort": 80,
                                "adminState": "enable",
                                "serverAddresses": [
                                    "10.1.20.100"
                                ]
                            }
                        ],
                        "monitors": [
                            "http"
                        ],
                        "class": "Pool"
                    },
                    
                    "compress_profile":{
                        "bufferSize": 128000,
                        "class": "HTTP_Compress"
                    }
                }
            }
        }
    }
```

2. Observe the following error response. Check Task Status ... Fails, constrains to Schema. Schema should adapt to BIG-IP:
```json
{
    "id": "7f952f89-552e-45a0-8e2f-389eb8c337f2",
    "results": [
        {
            "code": 422,
            "errors": [
                "/app_1/app_1/compress_profile/bufferSize: should be <= 32768"
            ],
            "declarationFullId": "",
            "message": "declaration is invalid"
        }
    ],
    "declaration": {}
}
```

### Expected Behavior
Permit values in AS3 permitted by BIG-IP.

### Actual Behavior
Schema enforces a lower max value than accepted on BIG-IP.

**Comment by RobCupples** [comment]
Thank you for reporting this. It is being tracked in our internal product backlog as AUTOTOOL-1626

**Comment by dstokesf5** [comment]
This has been resolved as part of the 3.21.0 release.
